R/matrix_form.R

```diff
@@ -167,10 +186,21 @@ disp_from_spans <- function(spans) {
   display
 }
 
+.fixup_cinfo <- function(col_df) {
+  if (is.null(col_df$par_extent)) {
+    col_df$par_extent <- 0L
+  }
+  if (is.null(col_df$node_class)) {
+    col_df$node_class <- "Column"
+  }
+  col_df
+}
```

Comment:
it seems this function is not used

Reply:
Hmm, surprising but appears to be true, I've taken it out of my local copy and will push